src/main/java/reposense/analyzer/FileInfoGenerator.java

```diff
@@ -22,7 +26,7 @@ public static FileInfo generateFileInfo(String repoRoot, String relativePath) {
                 lineNum += 1;
             }
         } catch (IOException ioe) {
-            ioe.printStackTrace();
+            logger.severe(LogsManager.getErrorDetails(ioe));
```

Comment:
Instead of doing this, have you tried this overload method?
https://docs.oracle.com/javase/7/docs/api/java/util/logging/Logger.html#log(java.util.logging.Level,%20java.lang.String,%20java.lang.Throwable)